src/test/java/com/microsoft/sqlserver/jdbc/datatypes/JSONFunctionTest.java

```diff
@@ -0,0 +1,1862 @@
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.CallableStatement;
+import java.sql.Clob;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Tag;
+
+import com.microsoft.sqlserver.jdbc.RandomUtil;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+import com.microsoft.sqlserver.testframework.Constants;
+
+@RunWith(JUnitPlatform.class)
+@DisplayName("Test Json Functions")
+@Tag(Constants.JSONTest)
+public class JSONFunctionTest extends AbstractTest {
+
+    @BeforeAll
+    public static void setupTests() throws Exception {
+        setConnection();
+    }
+    
+    private static final String JSON_FILE_PATH = "large_json.json";
+    private static String procedureName = RandomUtil.getIdentifier("testJsonProcedure");
+
+    /**
+     * Test ISJSON function with JSON.
+     * ISJSON -> Tests whether a string contains valid JSON.
+     */
+    @Test
+    @Tag(Constants.JSONTest)
+    public void testISJSON() throws SQLException {
+        String dstTable = TestUtils
+                .escapeSingleQuotes(AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("dstTable")));
+
+        try (Connection conn = DriverManager.getConnection(connectionString);) {
+            try (Statement dstStmt = conn.createStatement()) {
+
+                dstStmt.executeUpdate(
+                        "CREATE TABLE " + dstTable + " (testCol JSON);");
+
+                String validJson = "{\"key1\":\"value1\"}";
+                dstStmt.executeUpdate(
+                        "INSERT INTO " + dstTable + " (testCol) VALUES ('" + validJson + "')");
+
+                String select = "SELECT testCol, " +
+                        "ISJSON(testCol) AS isJsonValid " +
+                        "FROM " + dstTable;
+                try (ResultSet rs = dstStmt.executeQuery(select)) {
+                    assertTrue(rs.next());
+                    assertEquals(validJson, rs.getString("testCol"));
+                    assertEquals(1, rs.getInt("isJsonValid"));
+                }
+
+            } catch (Exception e) {
+                fail(e.getMessage());

```

Comment:
This catch (and all the others like it) are unnecessary. The uncaught Exception will provide better information in the test failure results.